.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - '6'
   - '5'
```

Comment:
I was thinking of doing that in 2 months! 😄 

Seems I side with @novemberborn here.